### PR TITLE
Enhance karmor sysdump to include logs from all kubearmor components with the label kubearmor-app

### LINKED ISSUE #441: `karmor sysdump` should contain logs from all pods with `kubearmor-app` label
### Description

Currently `karmor sysdump` command only takes logs from pods with the label `kubearmor-app=kubearmor` however for debugging purposes, logs from other components like kubearmor-relay-server, kubearmor-controller and kubeamor-operator might be required as well.

### Solution

Get the logs from all pods which have the `kubearmor-app` label so that we can get more insights while debugging.

### PR DESCRIPTION
**Pull request by Ishaanj18**
Changes Made:

- Fixes #441
-  Updated the label filter to include pods labeled "kubearmor-app", expanding the scope to gather logs from all relevant components.
-  Implemented a for loop to iterate through all containers within each pod, ensuring logs are collected comprehensively even from pods with multiple containers.

### CODE REVIEW

File: sysdump/sysdump.go [hunk 1]
```diff
@@ -97,34 +97,38 @@ func Collect(c *k8s.Client, o Options) error {
 	// KubeArmor Pod
 	errs.Go(func() error {
 		pods, err := c.K8sClientset.CoreV1().Pods("").List(context.Background(), metav1.ListOptions{
-			LabelSelector: "kubearmor-app=kubearmor",
+			LabelSelector: "kubearmor-app",
 		})
+
 		if err != nil {
 			fmt.Printf("kubearmor pod not found. (possible if kubearmor is running in process mode)\n")
```
Comment (rootxrishabh) [inline sysdump/sysdump.go:104]: Let's update the print statement to inform the user that it is checking all the KubeArmor pods.

**Comment by DelusionalOptimist** [comment]
Hey @Ishaanj18 can you address the reviews and update this PR so that tests pass?

**Comment by rootxrishabh** [comment]
@Ishaanj18, please squash the commits.

File: sysdump/sysdump.go [hunk 2]
```diff
@@ -97,34 +97,40 @@ func Collect(c *k8s.Client, o Options) error {
...
 			return nil
+		} else {
+			fmt.Print("Checking all pods labeled with kubearmor-app\n")
```
Comment (DelusionalOptimist) [inline sysdump/sysdump.go:107]: NIT: Please address the lint error here. Rest all looks great!

**Review by DelusionalOptimist — APPROVED** [review]
LGTM. Thanks @Ishaanj18! 🚀

**Review by rootxrishabh — APPROVED** [review]
LGTM!

**Pull request merged**